ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-master-postsubmits.yaml

```diff
@@ -12,7 +12,7 @@ postsubmits:
         - --artifact-dir=$(ARTIFACTS)
         - --give-pr-author-access-to-namespace=true
         - --secret-dir=/usr/local/e2e-secrets/azure
-        - --target=[output:stable:sync]
+        - --target=[output:stable:sync:logbridge]
```

Comment:
Interesting. Is this documented somewhere?

Reply:
This is target list. I know I needed to add sync to be promoted to another stage. We will need in the future enable our e2e to use this new logbridge image too, same as sync.

Reply:
Is this notation documented somewhere in the ci-operator docs?

Reply:
Not much :/ `https://github.com/openshift/release/tree/master/ci-operator#using-a-template`
I remember that it was not promoting sync If I didn't have this one...